tools/build_standard_commutations.py

```diff
@@ -214,32 +214,71 @@ def _simplify_commuting_dict(commuting_dict: dict) -> dict:
     return commuting_dict
 
 
+def format_entry(key, value):
+    """Generate a string for a commutation entry."""
+    with io.StringIO() as buf:
+        buf.write("(smallvec![")
+        for qubit in key:
+            if qubit is None:
+                buf.write("None, ")
+            else:
+                buf.write(f"Some(Qubit({qubit})), ")
+        buf.write(f"], {str(value).lower()})")
+        return buf.getvalue()
+
+
 def _dump_commuting_dict_as_python(
```

Comment:
This should probably be renamed
```suggestion
def _dump_commuting_dict_as_rust(
```